internal/services/controllers/workflows/queue.go

```diff
@@ -23,6 +23,10 @@ import (
 )
 
 func (wc *WorkflowsControllerImpl) handleWorkflowRunQueued(ctx context.Context, task *msgqueue.Message) error {
+
+	// TODO remove de dupes and fail if they are clashing
```

Comment:
Just making sure we address this TODO before merge

Reply:
added some dedupe logic here